medical_diagnostic_report/models/medical_diagnostic_report.py

```diff
@@ -117,6 +117,8 @@ def _get_states(self):
     )
     is_cancellable = fields.Boolean(compute="_compute_is_cancellable")
 
+    specialty_ids = fields.Many2many("medical.specialty", string="Specialties")
```

Comment:
Why?

Reply:
we were trying to replicate the tree view of the Sky

Reply:
👍🏽